Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -150,6 +150,21 @@ inline void ClearStrokeStyle() {
         return GStateStack().top();
     }
 
+    inline bool HasPath() {
+        return _impl.currentPath != nullptr;
+    }
+
+    inline CGMutablePathRef Path() {
+        if (!_impl.currentPath) {
+            _impl.currentPath.reset(CGPathCreateMutable());
+        }
+        return _impl.currentPath.get();
+    }
+
+    inline void SetPath(CGMutablePathRef path) {
```

Comment:
`unique_cf` takes care of this.

---
In reply to: [85814512](https://github.com/Microsoft/WinObjC/pull/1240#discussion_r85814512) [](ancestors = 85814512)